pom.xml

```diff
@@ -69,6 +69,7 @@
     <module>r2dbc-core</module>
     <module>r2dbc-mysql</module>
     <module>r2dbc-postgres</module>
+    <module>sqlserver</module>
```

Comment:
Should we consider stacking these into JDBC and r2dbc folders?

Reply:
That would be a good idea, especially if we plan to add SQL Server r2dbc support soon.